### ISSUE TITLE
Accuracy overwrites accuracy training curve

### ISSUE THREAD

**Issue by SamVanhoutte** [arcus-azure/arcus.azureml#75]
When performing classification with Keras and evaluating it through AzureML, there is a training curve that contains `accuracy` as metric, but we also overwrite the accuracy at the end with one value.  therefore, the initial accuracy values for all but last epochs is zero, which is not desired.

**solution:**
change names of training curve metrics
